apps/client/src/common/utils/mergeProjects.ts

```diff
@@ -0,0 +1,19 @@
+import { DatabaseModel } from 'ontime-types';
+
+import { getDb, patchData } from '../api/db';
+
+export async function mergeProjects(fileName: string, mergeKeys: object) {
```

Comment:
I think DataProvider already exposes a function to do this?

Reply:
yeah that is what the `patchPartialProjectFile` route uses but we need to create the partial object first